framework/decode/vulkan_ascii_consumer_base.h

```diff
@@ -109,22 +111,52 @@ class VulkanAsciiConsumerBase : public VulkanConsumer
 
   protected:
     template <typename ToStringFunctionType>
-    inline void WriteApiCallToFile(const ApiCallInfo&   call_info,
-                                   const std::string&   functionName,
-                                   util::ToStringFlags  toStringFlags,
-                                   uint32_t&            tabCount,
-                                   uint32_t             tabSize,
-                                   ToStringFunctionType toStringFunction)
+    inline void WriteApiCallToFile(const ApiCallInfo& call_info,
```

Comment:
You were right, @bradgrantham-lunarg. By putting more effort into this function, I could simplify code gen of functions that call it and the manual functions that do too.